### PR TITLE
README: Expand README.md

### PR DESCRIPTION
**Pull request by rjnn**
Add basic build instructions, and a short description of each load
generator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+
+The load generator can generate different "scalefactors" of data:
+scalefactor-1 is roughly 1gb in size of CSVs (translating into about
+~1.7gb in CockroachDB counting denormalizations), and scales linearly
+for larger scalefactors.
+
+
+## Rand
+
+Rand generates random data andm inserts it into CockroachDB. It
```
Comment (jordanlewis) [inline README.md]: nit: typo in and

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+# CockroachDB load generators
+
+This repository contains a few load generators for load testing
+CockroachDB (and on occasion, other databases).
+
+# Building
+
+`make build` will build all load generators. The binaries can be found
+in the individual directories (`kv/kv`, `tpcc/tpcc`,`'tpch/tpch`,
+`ycsb/ycsb`).
```
Comment (jordanlewis) [inline README.md]: plz include rand

**Review by jordanlewis — APPROVED** [review]
Thanks!

**Review by a-robinson — APPROVED** [review]
Thanks!

File: README.md (hunk 2 above)
Comment (rjnn) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (rjnn) [reply README.md]: Done.

**Pull request merged**